src/search/landmarks/landmark_factory_reasonable_orders_hps.cc

```diff
@@ -394,6 +397,19 @@ class LandmarkFactoryReasonableOrdersHPSFeature : public plugins::TypedFeature<L
                 "2004"));
 
         add_option<shared_ptr<LandmarkFactory>>("lm_factory");
+        /* TODO: Print warning in admissible heuristics when using
+            obedient-reasonable. */
+        // TODO: Should we adapt aliases to set this option to true?
+        add_option<bool>(
+            "use_obedient_reasonable",
+            "Choose whether obedient-reasonable orderings should also be "
+            "computed. Note that we are unsure whether and how they could be "
+            "used for optimal planning (i.e., in admissible heuristics). "
+            "Furthermore, our experiments suggest that they are not really "
+            "helpful for satisficing planning either, hence the default value "
+            "*false*.",
+            // FIXME: change default value to *false*!
+            "true");
```

Comment:
I wouldn't add this option at this stage just to next rip out the feature again. I'd either leave it unchanged in this issue and rip it out later, or rip it out now.

Reply:
I agree, that's wasted effort. We also have the `TODO` a few lines above for printing a warning when using obedient-reasonable orderings in admissible heuristics. I assume it's okay if I also remove that?

Reply:
Yes.